app/views/models/edit.html.erb

```diff
@@ -16,6 +16,9 @@
         <%= f.text_area :description, :class=>"form-control rich-text-edit", :rows => 5 -%>
       </div>
 
+      <%= render :partial => 'extended_metadata/extended_metadata_type_selection', :locals => {:f=> f,:resource => @model } %>
+      <%= render :partial => 'extended_metadata/extended_metadata_attribute_input', :locals => {:f=> f, :resource => @model } %>
+
```

Comment:
use the format `partial:` rather than `:partial=>` , and the same for other keys